main.go

```diff
@@ -42,15 +43,69 @@ func main() {
 	tc := oauth2.NewClient(ctx, ts)
 	client := github.NewClient(tc)
 
-	// Get the Pull Request details
+	// Fetch PR details
 	pr, _, err := client.PullRequests.Get(ctx, *owner, *repo, *prNumber)
 	if err != nil {
 		fmt.Printf("Error fetching PR details: %v\n", err)
 		os.Exit(1)
 	}
 
-	// Generate a review using GPT-4
```

Comment:
"The usage string refers to 'approve' and 'request-changes', which are no longer valid flags. Please update to reflect the current dry run option.